### ISSUE TITLE
Notice at end of install script, persistent issues post-install

### ISSUE THREAD

**Issue by randall-mcpherson** [ChrisTitusTech/Debian-titus#89]
Second install attempt. Can getting working OS both times, but keep getting weird notice that "The home dir/var/lib/usbmux you specified can't be accessed: No such file or directory" upon executing the install script, toward the end. Also a final message after saying the process "succeeded" indicates (in red) that "you need to be a member of the super use group to use me." If I reboot, it goes straight into the display manager (loggin) despite not having installed one (as per the youtube tutorial... did that get changed since the video was uploaded?) If I don't reboot, the terminal output remains in red for the entirety of the session. 

Upon reboot, various things feel "off." Most notably, ctl+f does not work, and the kitty terminal is set to a size much larger than my monitor (at least 2X for my 1080p monitor). Have to manually resize and reposition it every time the application is opened, and configuring most (but not all) default settings seem to reset upon application restart. Computer is an optiplex 5055 ryzen APU, if that matters.

**Comment by BrentKosior** [comment]
yeah I've tried multiple times on both VM's and bare metal and install doesn't work on either. Gets hung on installing like icons or something and then isn't able to boot.

**Comment by miczukow** [comment]
I have the same problem with the oversized kitty terminal, and arandr don't fix it in any resolution setting. Plus I don't see that top bar at all. Which log file shoud I check to track any install.sh errors?
